### DISCUSSION TITLE
Teleport to homes using numbers.

### DISCUSSION THREAD

**Discussion by goosBanny** [Zrips/CMI#6948] [Enhancement]
In EssentialsX, /home # teleports the player to a home. CMI should do the same. i.e. /home 1 would teleport the player to their first set home.

**Comment by mrfloris** [comment]
Players could just name their home "1" and then they can /home 1

**Reply by goosBanny** [reply]
True, but this would make it more difficult to keep track of where a home is. Perhaps an "alias" of sorts?

**Comment by MrCrazys** [comment]
What would happen if a player had a home called 1? I really don’t thionk this is necessary, and I think it would cause more problems than it would solve.
